src/test/java/org/jenkinsci/plugins/workflow/cps/PersistenceProblemsTest.java

```diff
@@ -245,7 +244,6 @@ public void inProgressNormal() throws Exception {
     }
 
     @Test
-    @Ignore
```

Comment:
These tests have been ignored since they were added in https://github.com/jenkinsci/workflow-cps-plugin/pull/223. They were unignored when the tests were duplicated over in https://github.com/jenkinsci/workflow-job-plugin/pull/104 (see https://github.com/jenkinsci/workflow-job-plugin/pull/104/commits/35b435d87ee77d96857f65c26b5207a299450781), so I think the fact that they are still ignored here was just an accident. Since I was running these tests anyway to test the logging changes, I went ahead and unignored them.